controllers/front/return.php

```diff
@@ -59,9 +65,13 @@ public function postProcess()
                 $orderStatusService->setPending($order);
             }
         } catch (SaferPayApiException $e) {
-            \PrestaShopLogger::addLog($e->getMessage());
+            $logger->debug(sprintf('%s - safe exception %s', self::FILE_NAME, $e->getMessage()), [
```

Comment:
If code comes to catch block it means exception was thrown make sure to log accordingly
            $logger->error('Failed to validate security token', [
                'context' => [],
                'exceptions' => ExceptionUtility::getExceptions($exception),
            ]);